libyul/backends/wasm/polyfill/Interface.yul

```diff
@@ -40,13 +42,15 @@ function chainid() -> z1, z2, z3, z4 {
 }
 
 function origin() -> z1, z2, z3, z4 {
-	eth.getTxOrigin(0:i32)
-	z1, z2, z3, z4 := mload_internal(0:i32)
+	memset(0:i32, 0:i32, 12:i32)
```

Comment:
These are not needed if you use mload_address, are they?

Reply:
These are not really needed. I removed these instructions.